_posts/en/pages/2016-01-15-lifecycle.md

```diff
@@ -16,44 +16,33 @@ This document describes the life-cycle of the Bitcoin Core software package rele
-
-Minor releases will be numbered 22.1, 22.2, 23.1, 23.2 etc.
+We will provide maintenance minor releases that fix bugs (security and otherwise) for each major release. These
+will be numbered 29.3, 30.1, etc. We will not introduce major new features in maintenance releases (besides consensus
+rules change, see below).
 
 ## Consensus rules
 
 Proposals to change consensus rules are always shipped first in maintenance versions such as 22.2, 23.1 etc. This makes it easier for enterprise users to assess and test the proposal because of its smaller changeset compared to a major release. It also allows users who follow a more conservative upgrade path to adopt consensus rule changes in a more timely manner.
 
 ## Maintenance period
 
-We maintain the major versions until their "Maintenance End". We generally maintain the current and previous major release.
-For example, if the current release is 23.0, then 22.0 is also considered maintained.
-Once 24.0 is released, then 22.0 would be considered at its "Maintenance End".
-As a major release ages, issues have to be increasingly critical to be backported to it, and an increasing amount or severity of issues is required to warrant a new minor release.
-Once software has reached the "Maintenance End" period, it will only receive critical security fixes until the End-of-Life (EOL) date.
-After EOL, users must upgrade to a later version to receive security updates, even though the community may provide fixes for critical issues on a best effort basis.
-Generally, it is recommended to run the latest maintenance release (point release) of the current or previous major version.
-
-Please note that minor versions get bugfixes, translation updates, and soft forks. Translation on [Transifex][bitcoin-transifex-link] is only open for the last two major releases.
+We always maintain the past three major versions released. Once a major version falls outside that window, it becomes
+"End of Life". The threshold for backporting a change to a past major version increases as it ages. For example, if the
+last major release is 30.0, then 29.x and 28.x are also considered maintained. Once 31.0 is released, then 28.x becomes
+"End of Life".
 
-For example, major version 22.0 was released on 2021-09-13 and we provided maintenance fixes (point releases) until 2022-12-14.
-Critical security issues would still be continued to be fixed until the EOL date of 2023-04-01.
-However, to take advantage of bug fixes, you would have to upgrade to a later major version.
+Major versions that are "End of Life" do not, as a general rule, receive security fixes. For more about our policy on
+security fixes, see our [security advisories](/en/security-advisories/) page. We recommend running the latest
+maintenance release (point release) of the most recent major version you are able to upgrade to.
```

Comment:
Previously, this is referred to as “minor release”. I would recommend using the same term to refer to the same concept:

```suggestion
maintenance release (minor release) of the most recent major version you are able to upgrade to.
```